web/src/recent_topics_util.js

```diff
@@ -1,25 +0,0 @@
-let is_rt_visible = false;
-
-export function set_visible(value) {
-    is_rt_visible = value;
-}
-
-export function is_visible() {
-    return is_rt_visible;
-}
-
-export function get_topic_key(stream_id, topic) {
-    return stream_id + ":" + topic.toLowerCase();
-}
-
-export function get_key_from_message(msg) {
-    if (msg.type === "private") {
-        // The to_user_ids field on a direct message object is a
-        // string containing the user IDs involved in the message in
-        // sorted order.
-        return msg.to_user_ids;
-    } else if (msg.type === "stream") {
-        return get_topic_key(msg.stream_id, msg.topic);
-    }
-    throw new Error(`Invalid message type ${msg.type}`);
-}
```

Comment:
Removing `throw new Error` is what I'm particularly not sure about - we may need caution when muting an error. Some paths to think about:
1. The function param `msg` might not be annotated with a comprehensive type (currently `msg: Recipient` whose type is only `private` or `stream`)
2. Error (if happens) should be reported earlier at the caller (i.e before entering `get_key_from_message` function)